source/manual/intro-to-govuk-docker.html.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Intro to Docker and govuk-docker
+section: Docker
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-07-01
+review_in: 6 months
+owner_slack: "#govuk-developers"
+---
+
+## Intro
+If you are new to Docker here's a quick intro in the context of the GOV.UK stack.
+
+
+### Install Docker
+
+*   Download Docker [https://www.docker.com/get-started](https://www.docker.com/get-started)
+*   Docker for Mac intro [https://docs.docker.com/docker-for-mac/](https://docs.docker.com/docker-for-mac/)
+
+### Video explaining the concepts in Docker
+
+[https://www.youtube.com/watch?v=i7yoXqlg48M](https://www.youtube.com/watch?v=i7yoXqlg48M) - first 30 minutes
+
+## Learning docker with content-publisher
+We'll run a few things here to get the [content-publisher] up and running in Docker. This is a convoluted example but it will help to explain the concepts involved.
+
+### Run a container from an image
+
+* Make sure you have docker running:
+
+```shell
+$mac docker --version
+Docker version 18.09.2, build 6247962
+```
+
+* run from the root of the [content-publisher] project:
+
+```shell
+$mac cd ~/govuk/content-publisher
+$mac docker run -it --rm ubuntu
```

Comment:
I wonder if we should skip the ubuntu example and just go straight for using the ruby image. It may be less confusing and/or time consuming (because each image will need to be downloaded).